### PR TITLE
Change default shortcuts for zooming in and out of graph

### PR DESCRIPTION
**Pull request by Quasar985**
### Prerequisites

- [x] Reviewed the [checklist](CHECKLIST.md)

- [x] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

Changed the default keyboard shortcuts for zooming in and zooming out to better suit Linux. `CTRL -` now zooms out and `CTRL =` now zooms in. The thought process being the physical keys have the plus and minus symbol on them.

### Alternate Designs

None.

### Why Should This Be In Core?

N/A

### Benefits

Default key binds now work on Linux

### Possible Drawbacks

None.

### Verification Process

1. Open consty
2. Open or create a graph with nodes in it.
3. Use `CTRL -` to zoom out
4. Use `CTRL =` to zoom in

### Applicable Issues

https://github.com/constellation-app/constellation/issues/2165

### CODE REVIEW

**Review by Delphinus8821 — APPROVED** [review]

**Review by antares1470 — COMMENTED** [review]
There might be an entry in `whatsnew.txt` that should be updated

File: CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 <p>A dropdown list has been added to the top of the view to select how many Conversation bubbles to display per page.</p>
 
 == 2024-03-08 More Ways to Zoom In and Out
-<p>Keyboard shortcuts have been added for zooming in and out of the graph view (CTRL-ALT-UP and CTRL-ALT-DOWN respectively).</p>
+<p>Keyboard shortcuts have been added for zooming in and out of the graph view (CTRL - and CTRL = respectively).</p>
```
Comment (antares1470) [inline CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt:19]: I think its the other way around

**Comment by antares1470** [comment]
@Quasar985 just had an idea, if we are using - and = for zoom out and in, would it also make sense to include shortcuts using - and + on the num pad (i.e. Ctrl-Subtract and Ctrl-Add)?

**Comment by Quasar985** [comment]
> @Quasar985 just had an idea, if we are using - and = for zoom out and in, would it also make sense to include shortcuts using - and + on the num pad (i.e. Ctrl-Subtract and Ctrl-Add)?

I've added these as extra shortcuts

**Review by antares1470 — COMMENTED** [review]
It appears the Ctrl-Add and Ctrl-Subtract shortcuts don't work for me. When looking at the keymap, it appears the actions are only applied to one shortcut each

**Comment by Quasar985** [comment]
> It appears the Ctrl-Add and Ctrl-Subtract shortcuts don't work for me. When looking at the keymap, it appears the actions are only applied to one shortcut each

That's odd. The extra shortcuts should be listed in the keymap. They should be called something along the lines of "Zoom In (Additional)".
Just checking, have you pulled the latest version?

**Comment by antares1470** [comment]
> That's odd. The extra shortcuts should be listed in the keymap. They should be called something along the lines of "Zoom In (Additional)".
> Just checking, have you pulled the latest version?

So I thought I had as I had just checked out the branch for the first time...but apparently it wasn't checked out with the most recent commit...we're all good now

**Review by antares1470 — APPROVED** [review]

**Pull request merged**